### DISCUSSION TITLE
Aggregate report for multiple .jmx files?

### DISCUSSION THREAD

**Discussion by ftchen93** [nonealexq/jmeter-allure-reporting#15] [General]
Hi author, this repo is great since i'm finding a way to generate report for jmeter API test.
However, in my scenario one test case is stored as one jmeter .jmx file hence i would like to ask if there is a way to show all results in just one allure report? And I realise for one request in jmeter it will show as one suite in allure report, is that a way to group them per test case? thanks!

**Comment by nonealexq** [comment]
Hi!
The entire report is generated from allure-results/.result
If you run n-jmx files, and then generate a report from this allure-results folder - everything will be in one report
